src/domains/creator/StudioContinuityPanel.tsx

```
@@ -76,48 +177,176 @@ function resolveSceneLabel(
+
+  const report = useMemo(
+    () =>
+      inspectStudioQuality({
+        pages,
+        continuityIssues: issues,
+        openCommentCount,
+        supplementalIssues:
+          rasterInspection?.status === "complete" ? rasterInspection.issues : [],
+      }),
+    // scanEpoch intentionally retries browser font measurement after fonts/resources settle.
+    [issues, openCommentCount, pages, rasterInspection, scanEpoch]
```

ESLint

[error] React Hook useMemo has an unnecessary dependency: 'scanEpoch'. Either exclude it or remove the dependency array.

(react-hooks/exhaustive-deps)